dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/AccessTokenController.java

```diff
@@ -74,20 +72,18 @@
     @PostMapping(value = "/create")
     @ResponseStatus(HttpStatus.CREATED)
     @ApiException(CREATE_ACCESS_TOKEN_ERROR)
-    @AccessLogAnnotation(ignoreRequestArgs = "loginUser")
-    public Result createToken(@ApiIgnore @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
-                              @RequestParam(value = "userId") int userId,
+    @AccessLogAnnotation()
+    public Result createToken(@RequestParam(value = "userId") int userId,
                               @RequestParam(value = "expireTime") String expireTime,
                               @RequestParam(value = "token") String token) {
 
-        Map<String, Object> result = accessTokenService.createToken(loginUser, userId, expireTime, token);
+        Map<String, Object> result = accessTokenService.createToken(AuthUtil.user(), userId, expireTime, token);
```

Comment:
I think the user parameters in the `Service` can also be removed

Reply:
yes, but too many methods to modify, so I'll remove it in another pr

Reply:
and this change will impact all test cases of service.